modules/driver_utilities.py

```diff
@@ -0,0 +1,113 @@
+import os
+import sys
+from pathlib import Path
+from visread import *
+
+def get_library_path(module_name: str) -> str:
+    """Determines the correct library path based on platform and module name.
+
+    Args:
+        module_name: Name of the module
+
+    Returns:
+        str: Path to the module library for the current platform.
+
+    Raises:
+        ValueError: If the current platform is not supported.
+        SystemExit: If on Windows and the DLL cannot be found in expected locations.
+    """
+    # Map module names to their library base names
+    module_map = {
+        "aerodyn": "libaerodyn_inflow_c_binding",
+        "hydrodyn": "libhydrodyn_c_binding",
+        "inflowwind": "libifw_c_binding",
+        "moordyn": "libmoordyn_c_binding"
+    }
+    basename = module_map.get(module_name.lower(), f"lib{module_name}_c_binding")
+    build_path = Path("..").joinpath(*[".."] * 4, "build")
```

Comment:
I would prefer we didn't hard code this to the `build` directory.  But I think that is what we already had.